index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>FCC: Survey Forms</title>
+    <link rel="stylesheet" href="./styles/style.css" />
+  </head>
+  <body>
+    <main>
+      <h1>freeCodeCamp Survey Form</h1>
+      <p id="para">Thank you for taking time to help us improve the platform</p>
```

Comment:
I think this should be a section different from the forms section. try putting them into different sections for better readablity.

Reply:
Good job 👍 in identifying the proper use of semantic tags.